device_tags_python/Dockerfile.template

```diff
@@ -0,0 +1,25 @@
+# base-image for python on any machine using a template variable,
+# see more about dockerfile templates here: https://www.balena.io/docs/learn/develop/dockerfile/
+FROM balenalib/%%BALENA_ARCH%%-debian-python:3-bookworm
+
+# use `install_packages` if you need to install dependencies,
+# for instance if you need git, just uncomment the line below.
+# RUN install_packages git
+
+# Set our working directory
+WORKDIR /usr/src/app
+
+# Copy requirements.txt first for better cache on later pushes
+COPY requirements.txt requirements.txt
+
+# pip install python deps from requirements.txt on the resin.io build server
+RUN pip install -r requirements.txt
+
+# This will copy all files in our root to the working  directory in the container
+COPY . ./
+
+# Enable udevd so that plugged dynamic hardware devices show up in our container.
+# ENV UDEV=1
+
+# main.py will run when container starts up on the device
+CMD ["python","-u","src/app.py"]
```

Comment:
as far as I understand this will get a flask dev server up rather than a wsgi run with gunicorn See: https://flask.palletsprojects.com/en/stable/deploying/gunicorn/

are we ok with this?

Reply:
I don't really know :)
I just based this on our https://github.com/balena-io-examples/balena-python-hello-world